config/config_test.go

```diff
@@ -63,14 +65,31 @@ func TestInvalidConfigs(t *testing.T) {
 	for k, v := range invalidConfigs {
 		t.Run(k, func(t *testing.T) {
 			filename := mkConfig(t, v)
-			_, err := ReadConfigFile(filename, "", "")
+			_, err := ReadConfigFile(filename, "", "", "")
 			if err == nil {
 				t.Fatal("this config must generate an error")
 			}
 		})
 	}
 }
 
+func TestConfigDir(t *testing.T) {
+	colorGreen := "\033[32m"
```

Comment:
Id prefer we keep the test output formatting standard. Also this will make jenkins build output a bit messy.